webpack.config.js

```diff
@@ -12,18 +14,19 @@ const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const devServerProxyTarget
     = process.env.WEBPACK_DEV_SERVER_PROXY_TARGET || 'https://alpha.jitsi.net';
 
-const analyzeBundle = process.argv.indexOf('--analyze-bundle') !== -1;
-const detectCircularDeps = process.argv.indexOf('--detect-circular-deps') !== -1;
-
-const minimize
-    = process.argv.indexOf('-p') !== -1
-        || process.argv.indexOf('--optimize-minimize') !== -1;
-
 /**
  * Build a Performance configuration object for the given size.
  * See: https://webpack.js.org/configuration/performance/
+ *
+ * @param {Object} options - options for the bundles configuration.
+ * @param {boolean} options.analyzeBundle - whether the bundle needs to be analyzed for size.
+ * @param {boolean} options.minimize - whether the code should be minimized or not.
+ * @param {number} size - the size limit to apply.
+ * @returns {Object} a performance hints object.
  */
-function getPerformanceHints(size) {
+function getPerformanceHints(options, size) {
```

Comment:
calls to this only pass the size parameter

Reply:
Opsie, nice catch!